gcp_variant_transforms/testing/testdata_util.py

```diff
@@ -22,10 +22,11 @@
 
 
 def get_full_file_path(file_name):
-  """Returns the full path of the specified ``file_name`` from ``testdata``."""
-  return os.path.join(os.path.dirname(__file__), 'testdata', file_name)
+  """Returns the full path of the specified ``file_name`` from ``data``."""
+  return os.path.join(
+      os.path.dirname(__file__), '..', 'testing', 'data', 'vcf', file_name)
 
 
 def get_full_dir():
-  """Returns the full path of the  ``testdata`` directory."""
-  return os.path.join(os.path.dirname(__file__), 'testdata')
+  """Returns the full path of the  ``data`` directory."""
+  return os.path.join(os.path.dirname(__file__), '..', 'testing', 'data', 'vcf')
```

Comment:
can this just be os.path.join(os.path.dirname(__file__), 'data', 'vcf'? (without the ../testing)?

Reply:
Good point. I think in Beam this was necessary since the function was written in the io folder.